pkg/sql/logictest/testdata/logic_test/timestamp

```diff
@@ -192,3 +192,14 @@ query R
 SELECT extract(timezone FROM '2001-01-01 13:00:00+01:15'::timestamptz)
 ----
 10800
+
+
+# Test timezone() 
```

Comment:
nit: this line is a little misleading. maybe use `subtest regression_41776` instead? (no comment)